helpers.go

```diff
@@ -26,7 +26,7 @@ func goType(t TypeInfo) reflect.Type {
 		return reflect.TypeOf(*new(string))
 	case TypeBigInt, TypeCounter:
 		return reflect.TypeOf(*new(int64))
-	case TypeTimestamp:
+	case TypeTimestamp, TypeTime:
```

Comment:
This is not relevant after #1286, please remove

Reply:
@alourie Resolved, please check.